pkg/virt-controller/watch/ovm.go

```diff
@@ -466,7 +464,7 @@ func (c *OVMController) addVirtualMachine(obj interface{}) {
 			log.Log.Object(vm).Errorf("Cannot parse key of OVM: %s for VM: %s", ovm.Name, vm.Name)
 			return
 		}
-		log.Log.Object(vm).Infof("VirtualMachine created bacause %s was added.", vm.Name)
+		log.Log.Object(vm).Infof("VirtualMachine created because %s was added.", vm.Name)
```

Comment:
Please give that a higher verbosity and also change the text: e.g. "Creation for VirtualMachine observed".